### DISCUSSION TITLE
LIVESTREAM 9AM EST Saturday Nov 6, 2021: Live demos and answering any questions!

### DISCUSSION THREAD

**Discussion by jlongster** [actualbudget/releases#24] [Announcements]
If you're new to Actual, welcome! I'd love to sit down with you all and walk you through the app and answer any questions you have. Want to know how to handle credit cards? How to use schedules? How to create custom rules? I'm happy to walk through any part of the app!

I can also answer any general questions you have, such as the roadmap for actual, how end-to-end encryption works, and things like that.

Tomorrow at 9AM EST I'll be doing a livestream for about 1 hour. Please stop by and drop any questions you have!

**You can go ahead and ask questions by posting as a reply below. If the same question has been asked, please vote for it instead of duplicate it! I will answer question in order of highest votes.**

Watch here: https://www.twitch.tv/jlongster

**Comment by nathan-isaac** [comment]
I'd definitely like to see how end to end encryption works.

**Comment by AlejandroCatalinaF** [comment]
Is it possible to set goals per category? So, if I have an scheduled transaction on December 2021 for a given category, I'd like my budget to reflect that and tell me I should budget the corresponding proportional amount for the current month.

**Comment by ellgon** [comment]
Hello James

~~I am configuring all my scheduled transactions from Ynab and I do not understand really well how them works.
My main question is regarding the status. I have 6 scheduled transactions so far. Two of them appear as Paid while all the others appear as Scheduled. All transactions have a linked transaction for the current month and are configured with the same options.~~

~~What does the Status mean? Is there any way to change/update it manually?~~

**Reply by ellgon** [reply]
I answer myself, I made a mistake in the dates... so you can skip my question ;)

**Comment by bzupnick** [comment]
Is there a chance Actual will be open sourced? As a current YNABer I can attest to the enthusiasm and evangelism that, if brought to Actual, some developers (myself included) would love to help out and work on issues and features.

**Comment by makenova** [comment]
Will you add features for shared accounts? 
Discussion started over here: #23 

repeating here for livestream.
